app/Containers/VisibleToDoList.jsx

```diff
@@ -1,5 +1,5 @@
 import { connect } from 'react-redux';
-import { toggleToDo, removeTodo } from '../store/Actions/index.js';
+import { toggleToDo, removeTodo } from '../store/Actions/index';
```

Comment:
action and reducer folder should not inside store. after all, they are not part of store